mdpow/fep.py

```diff
@@ -252,13 +252,19 @@ def load(cfg, section):
         cfg_get = {float: cfg.getfloat,
                    int: cfg.getint,
                    str: cfg.getstr,    # literal strings, no conversion of None (which we need for the MDP!)
+                                       # CHECK: THIS IS LIKELY NOT GUARANTEED ANYMORE since getstr == get
                    list: cfg.getarray  # numpy float array from list
                    }
         def getter(type, section, key):
+            value = cfg_get[type](section, key)
             try:
-                return cfg_get[type](section, key)
-            except NoOptionError:
-                return None
+                # for empty list [] and array array([])
+                if len(value) == 0:
+                    value = None
```

Comment:
We need a test for `fep` for the case `len(value) == 0`.

Reply:
yes... btw, both you and @ALescoulie are more than welcome to say "this needs a test" and make it a blocking review, I won't feel offended.

Reply:
tested now